packages/far/test/test-marshal-far-obj.js

```diff
@@ -5,6 +5,10 @@ import { q } from '@endo/errors';
 
 import { Far, passStyleOf, getInterfaceOf } from '../src/index.js';
 
+/**
+ * @import {PassStyled} from '@endo/pass-style';
```

Comment:
Would I be right to assume that someday `eslint` will catch up with this notation and demand spaces inside the curlies?